app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -322,7 +324,7 @@ else if (locationManager.getLastKnownLocation(LocationManager.NETWORK_PROVIDER)!
 			lon = -1;
 		}
 		try{
-			mCamera = Camera.open();}catch(Exception e){
+			mCamera = Camera.open(PhimpMe.camera_use);}catch(Exception e){
```

Comment:
@Abhi2424shek It is within a try catch block. @duchuyctlk wants me to put catch clause in the next line but it is not related to my issue so I left it.